package/tests/testthat/_snaps/stable/tables-efficacy-rbmit01.md

```diff
@@ -13,24 +13,24 @@
           Relative Reduction (%)                                     5.7%      
           p-value (RBMI)                                            0.8932     
       5                                                                        
-        Adjusted Mean (SE)                   -4.251 (0.647)     -2.805 (0.644) 
-          95% CI                            (-5.528, -2.974)   (-4.077, -1.533)
-        Difference in Adjusted Means (SE)                       1.446 (0.914)  
-          95% CI                                               (-0.358, 3.250) 
-          Relative Reduction (%)                                    -34.0%     
-          p-value (RBMI)                                            0.1155     
+        Adjusted Mean (SE)                   -4.249 (0.654)     -2.813 (0.651) 
+          95% CI                            (-5.540, -2.957)   (-4.100, -1.526)
+        Difference in Adjusted Means (SE)                       1.436 (0.922)  
+          95% CI                                               (-0.385, 3.256) 
+          Relative Reduction (%)                                    -33.8%     
```

Comment:
These changes are really unexpected. Any idea about why? @shajoezhu @edelarua @ayogasekaram

Reply:
@Melkiades that's one of the tables that generates different values randomly depending on if it's run locally/in validated/non-validated integration tests.

If I generate this table locally, I get a third (different) set of values here:
```
5                                                                        
  Adjusted Mean (SE)                   -4.250 (0.647)     -2.814 (0.639) 
    95% CI                            (-5.527, -2.972)   (-4.076, -1.551)
  Difference in Adjusted Means (SE)                       1.436 (0.913)  
    95% CI                                               (-0.367, 3.238) 
    Relative Reduction (%)                                    -33.8%     
    p-value (RBMI)                                            0.1177     
```

Reply:
ahah ok it is one of our magic tables that depend on {tern.rbmi}. We need to understand what is going on there